docs/guides/custom-publisher-integration.md

```diff
@@ -29,8 +29,8 @@ The following are the options available for publishers to integrate with UID2:
 
 - Client UID2 SDK for JavaScript (see [UID2 SDK for JavaScript Reference Guide](../sdks/client-side-identity.md)), with [UID2 SDK for Java](../sdks/uid2-sdk-ref-java.md) on the server.
 - Client UID2 SDK for JavaScript (see [UID2 SDK for JavaScript Reference Guide](../sdks/client-side-identity.md)), with custom server code.
```

Comment:
your PR is fine but reading the above 2 lines are a bit confusing to me - i'm not sure what the above 2 means - is it saying the publisher needs to generate tokens on server-side using UID2 SDK for Java or custom server codes (i think it should be plural?) and then use JS SDK on the client-side to store the token and retrieve it for bidstream use. 

If this is the case, then shouldn't mentioning the Client-Server/Server-side integration below enough so we can remove the 2 lines above? 

the 2 lines above were from aug 2023 before we wrote up the client-server/server-side integration guides so i suspect the 2 lines above are just outdated now.


cc @jon8787 @lionell-pack-ttd for feedback.

Reply:
I agree - those first two options aren't needed any more.

Reply:
ok sounds like we are in-sync here but see the comment below on what to edit